Tasks/Maven/CodeAnalysis/mavenpmd.ts

```diff
@@ -28,6 +28,10 @@ export function collectPmdOutput(rootDir:string) : AnalysisResult {
 
     var pmdXmlFilePath = path.join(rootDir, 'target', 'pmd.xml');
     result = collectPmdXml(result, pmdXmlFilePath);
+    if (!result)
```

Comment:
Additional defensive coding was required here and in some other places, since the SQ plugin creates target folders that do not have pmd.xml files.